vendor/three/DeviceOrientationControls.js

```diff
@@ -1,116 +1,131 @@
-/**
- * @author richt / http://richt.me
- * @author WestLangley / http://github.com/WestLangley
- *
- * W3C Device Orientation control (http://w3c.github.io/deviceorientation/spec-source-orientation.html)
- *
- * originally from https://github.com/mrdoob/three.js/blob/r93/examples/js/controls/DeviceOrientationControls.js
- */
-import * as THREE from 'three';
+import {
+  Euler,
+  EventDispatcher,
+  MathUtils,
+  Quaternion,
+  Vector3
+} from '../../node_modules/three/build/three.module.js';
```

Comment:
I think you can just import `from 'three'`?

I'm looking at making threejs a peer dependency and that should help a good deal with three shaking?